app/helpers/locale_ninja/locale_helper.rb

```diff
@@ -16,7 +16,7 @@ def self.locales(github_service, branch: 'translations')
       github_service.locale_files_path(branch:).map { |path| path.scan(/\w+(?=\.yml)/).first }.uniq
     end
 
-    def self.locales_count(github_service, branch: 'translations')
+    def self.locales_count(github_service, branch = 'translations')
```

Comment:
why shouldn't this function use branch as a keyword argument ? I believe we should always give branch argument the same way